Bit-Manipulation/CountSetBits.js

```diff
@@ -0,0 +1,24 @@
+// author: Smit Gabani
+// https://github.com/smitgabani
+
+/**
+ * explanation link: https://iq.opengenus.org/brian-kernighan-algorithm/
+ * The below algorithm is called as Brian Kernighan's algorithm
```

Comment:
Poor comment. Should read something like "Brain Kernighan's Algorithm for Counting Set Bits" instead.